ateam_common/include/ateam_common/status.hpp

```diff
@@ -17,7 +17,7 @@
 // LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
-
+#pragma once
```

Comment:
Redundant pragma once when we already have the header guards below?

Reply:
Will remove